package.json

```diff
@@ -58,13 +60,13 @@
     "eslint-config-commercetools": "^6.0.0",
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-react": "^7.0.1",
+    "eslint-plugin-react": "^7.5.1",
```

Comment:
Why do we need react and jsx?

Reply:
Hmm.. I thought we need it because of aribnb-eslint but it seems like it is not needed anymore .. I will remove it

Reply:
Removed here 8e56cf3